web/src/styles/scss/dialog.scss

```diff
@@ -3,3 +3,10 @@
     border-top: none;
   }
 }
+.ds-u-left-border-warning{
+  border-left: 15px solid orange
```

Comment:
I pulled this hex color from Figma `#FDB81E` which might be a little different than the default orange. Maybe not though? ha

Reply:
fixed.  I thought I had done that but it got reverted somehow.